src/peer_sourcehash.cc

```diff
@@ -23,8 +23,9 @@
 
 #define ROTATE_LEFT(x, n) (((x) << (n)) | ((x) >> (32-(n))))
 
-static int n_sourcehash_peers = 0;
-static CachePeer **sourcehash_peers = nullptr;
+/// sourcehash peers ordered by their sourcehash weight
+static SelectedCachePeers TheSourceHashPeers;
```

Comment:
No new globals, please. This case may be interpreted as subject of that new policy because we are changing the old global _name_ and, hence, all its users.

I would convert the CARP storage as well, in this PR, but that is your call.